backend/src/routes/admin-routes.ts

```
@@ -384,21 +385,24 @@ const adminRoutes: FastifyPluginAsync = async (server) => {
-      if (!epVersion || (epVersion !== EPVersions.EPV_06 && epVersion !== EPVersions.EPV_07)) {
+      if (!epVersion || (epVersion !== EPVersions.EPV_06 && epVersion !== EPVersions.EPV_07 && epVersion !== EPVersions.EPV_08)) {
         return reply.code(ReturnCode.FAILURE).send({error: ErrorMessage.INVALID_EP_VERSION});
       }
 
-      const isEp06 = epVersion === EPVersions.EPV_06;
-
       const apiKeyEntity: APIKey | null = await server.apiKeyRepository.findOneByApiKey(apiKey);
       if (!apiKeyEntity) return reply.code(ReturnCode.FAILURE).send({ error: ErrorMessage.INVALID_API_KEY });
 
-      let verifyingPaymasters;
+      let verifyingPaymasters, supportedEPs;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)